lib/spec.js

```diff
@@ -78,210 +78,338 @@ rules = {
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson\" target=_blank><code>package.json</code> documentation</a> to see which properties are required and which are recommended."
     },
     "GS010-PJ-AUT-EM-VAL": {
         "level": "error",
-        "rule": "package.json <code>author.email</code> must be valid."
+        "rule": "<code>package.json</code> property <code>\"author.email\"</code> must be valid.",
+        "details": "The property <code>\"author.email\"</code> in your <code>package.json</code> file must a valid email. E.g. <code>{\"author\": {\"email\": \"hello@example.com\"}}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson#section--author-\" target=_blank><code>package.json</code> documentation</a> for further information."
     },
     "GS010-PJ-CONF-PPP": {
         "level": "recommendation",
-        "rule": "package.json <code>config.page_per_post</code> is recommended. Otherwise, it falls back to 5."
+        "rule": "<code>package.json</code> property <code>\"config.page_per_post\"</code> is recommended. Otherwise, it falls back to 5.",
+        "details": "Please add <code>\"posts_per_page\"</code> to your <code>package.json</code>. E.g. <code>{\"config\": { \"posts_per_page\": 5}}</code>.<br>" +
+                   "If no <code>\"posts_per_page\"</code> property is provided, Ghost will use its default setting of 5 posts per page.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson#section--config-posts_per_page-\" target=_blank><code>package.json</code> documentation</a> for further information."
     },
     "GS010-PJ-CONF-PPP-INT": {
         "level": "error",
-        "rule": "package.json <code>config.page_per_post</code> should be integer."
+        "rule": "<code>package.json</code> property <code>\"config.page_per_post\"</code> must be integer.",
+        "details": "The property <code>\"config.page_per_post\"</code> in your <code>package.json</code> file must be integer. E.g. <code>{\"config\": { \"posts_per_page\": 5}}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson#section--config-posts_per_page-\" target=_blank><code>package.json</code> documentation</a> for further information."
     },
     "GS010-PJ-AUT-EM-REQ": {
         "level": "error",
-        "rule": "package.json <code>author.email</code> is required."
+        "rule": "<code>package.json</code> property <code>\"author.email\"</code> is required.",
+        "details": "Please add the property <code>\"author.email\"</code> to your <code>package.json</code>. E.g. <code>{\"author\": {\"email\": \"hello@example.com\"}}</code>.<br>" +
+                   "Check the <a href=\"https://themes.ghost.org/docs/packagejson\" target=_blank><code>package.json</code> documentation</a> to see which properties are required and which are recommended."
```

Comment:
Possibly worth explaining about this property?

Basically, it is required so that themes which are distributed (either free or paid) have a method of contacting the author so users can get support and more importantly so that we can reach out about breaking changes and security updates.

Also worth explaining that the package.json file is NOT accessible when uploaded to a blog so if the theme is only uploaded to a single blog, no one will see this email address.